packages/nimble-components/src/menu/index.ts

```diff
@@ -11,10 +12,51 @@ declare global {
     }
 }
 
+declare type ItemsChangeHandler = (
+    oldValue: HTMLElement[] | undefined,
+    newValue: HTMLElement[]
+) => void;
+
 /**
  * A nimble-styled menu
  */
-export class Menu extends FoundationMenu {}
+export class Menu extends FoundationMenu {
+    private readonly baseItemsChanged: ItemsChangeHandler;
+
+    private readonly mutationObserver: MutationObserver = new MutationObserver(
+        mutations => {
+            for (const mutation of mutations) {
```

Comment:
> we have to jump through a couple hoops to override/call them. I think this is still preferable to copying the whole class implementation over and modifying it.

I'm curious what it would look like to copy the class and modify it. How big of a change would that modification be. The current approach requires `as` casts and `eslint-disable` statements, which doesn't seem ideal.

Reply:
I forked the Menu class from FAST and fixed all the lint issues. If you prefer the way I had it before, I can revert that commit.

Reply:
I believe your recent change to fork the Menu class is more aligned with our strategy moving forward. The FAST tests need to be forked as well.